src/emulator/modules/SceGxm/src/gxm.h

```diff
@@ -65,6 +65,8 @@ struct SceGxmContext {
     emu::SceGxmColorSurface color_surface;
     const SceGxmVertexProgram *vertex_program = nullptr;
     GLObjectArray<1> texture;
+    bool isInScene = false;
```

Comment:
Is it possible we can infer this from the color_surface field? It would be nice to not have extra fields that need kept in sync. There could be a helper function `is_in_scene` that derives it from the Context state.